sdks/python/apache_beam/coders/coders.py

```diff
@@ -420,6 +423,31 @@ def __hash__(self):
 Coder.register_structured_urn(common_urns.coders.BYTES.urn, BytesCoder)
 
 
+class BooleanCoder(FastCoder):
+  def _create_impl(self):
+    return coder_impl.BooleanCoderImpl()
+
+  def is_deterministic(self):
+    return True
+
+  def to_type_hint(self):
+    return bool
+
+  def as_cloud_object(self, coders_context=None):
+    return {
+        '@type': 'kind:bool',
+    }
```

Comment:
I'm not sure what this value should be.

Reply:
You shouldn't have to change as_cloud_object. That method is a Dataflow-specific thing hacked into the coders. Since Dataflow doesn't need to understand it, there's no need for anything custom.

Reply:
Ok, so I should delete this before it gets merged?

Reply:
ok, removed.